docs/mp/testing.adoc

```diff
@@ -36,25 +47,77 @@ include::{rootdir}/includes/dependencies.adoc[]
 </dependency>
 ----
 
-== Usage - default
+== Usage
 A test can be annotated with `io.helidon.microprofile.tests.junit5.HelidonTest` annotation to mark it as a
 CDI test. This annotation will start the CDI container before any test method is invoked, and stop it after
 the last method is invoked. This annotation also enables injection into the test class itself.
 
+=== Usage - per method CDI container
+A test can be annotated as follows:
```

Comment:
When a test is annotated with `@HelidonTest(resetPerTest = true)`, the behavior will change as follows:

Reply:
Fixed)